test/regress/regress0/print_lambda.cvc

```diff
@@ -1,7 +1,9 @@
 % SCRUBBER: sed -e 's/f : (INT) -> INT = (LAMBDA(.*:INT): 0);$/f : (INT) -> INT = (LAMBDA(VAR:INT): 0);/'
 % COMMAND-LINE: --produce-models
 % EXPECT: sat
+% EXPECT: MODEL BEGIN
 % EXPECT: f : (INT) -> INT = (LAMBDA(VAR:INT): 0);
```

Comment:
I am not completely sure CVC4 will always generate this value for `f`. It is definitely not the only possibility for it.
Maybe add a much simpler test, something like `x=5` over the integers where `x` is the only variable. Then there is a single model.

Reply:
Interesting, I didn't notice that. So how the old code can pass this test case?

Reply:
i guess CVC4 always returns this model, but you are right in the sense that Andres' and my concerns apply to the situation before this PR... So I guess I would be OK with leaving this test as is.

Reply:
new one added